src/cars/car.controller.ts

```diff
@@ -7,63 +7,47 @@ import {
   Patch,
   Delete,
   UseGuards,
-  Query,
 } from '@nestjs/common';
 import { AuthGuard } from '@nestjs/passport';
-import { User } from 'src/auth/entities/user.entity';
-import { GetUser } from 'src/auth/get-user-decorator';
+import { GetUser } from 'src/auth/decorator/get-user.decorator';
+import { UserCarOwnershipGuard } from 'src/auth/guard/user-car-ownership.guard';
+
 import { CarsService } from './car.service';
-import { CreateCarDto } from './dto/create-car.dto';
-import { SearchCarDto } from './dto/search-car.dto';
-import { Car } from './entities/car.entity';
+import { CarDto } from './dto';
 
 @Controller('cars')
-@UseGuards(AuthGuard())
+@UseGuards(AuthGuard('jwt'), UserCarOwnershipGuard)
```

Comment:
The `UserCarOwnershipGuard` should be only used for `@Patch('/:numberPlate')` and `@Delete('/:numberPlate')`.